src/js/claims-status/components/appeals-v2/Alerts.jsx

```diff
@@ -23,7 +23,7 @@ Alerts.propTypes = {
   alerts: PropTypes.arrayOf(PropTypes.shape({
     type: PropTypes.string.isRequired,
     date: PropTypes.string,
```

Comment:
`date` wasn't in the Swagger docs; does it still make sense here instead of inside `details`?

Reply:
Oh hmm, I could've sworn it was in there but it definitely isn't. I think I must've pulled the date property from `events`